server/routes/userRoutes.js

```diff
@@ -17,6 +19,8 @@ router.get("/profile", isAuthenticated, getUserProfile);
 router.get("/me", isAuthenticated, getUserProfile);
 router.patch("/update", isAuthenticated, updateUser);
 router.patch("/addcompany", isAuthenticated, addCompany);
+router.patch("/likemention", isAuthenticated, likeMention);
+router.patch("/unlikemention", isAuthenticated, unlikeMention);
```

Comment:
Maybe instead of creating two new routes for liking and unliking you can re-use the update route and perform the necessary operations in the likeMentions and unlikeMentions actions.

Reply:
Fair point ! I just made some changes